main.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	yaml "gopkg.in/yaml.v2"
+	"kcl-lang.io/kpm/pkg/client"
+	"kcl-lang.io/kpm/pkg/constants"
+	"kcl-lang.io/kpm/pkg/opt"
+	"oras.land/oras-go/v2"
+)
+
+const (
+	DefaultOciRegistry = "ghcr.io"
+	DefaultOciRepo     = "kcl-lang"
+	Homepage           = "KCL homepage"
+	HomepageLink       = "https://kcl-lang.io/"
+	Repo               = "KCL repo"
+	RepoLink           = "kcl-lang.io"
+	KclEmail           = "kcl-lang.io@domainsbyproxy.com"
+	KclName            = "kcl-lang"
+	README             = "README.md"
+
+	MdFlagPackageName = "<package_name>"
+	MdFlagPackageTag  = "<package_tag>"
+)
+
+type Metadata struct {
```

Comment:
Can we refer the definition from the artifact hub repo or add documents on this structure.